dm/pkg/schema/tracker_test.go

```diff
@@ -775,7 +775,7 @@ func (s *trackerSuite) TestGetDownStreamIndexInfo(c *C) {
 
 	mock.ExpectQuery("SHOW CREATE TABLE " + tableID).WillReturnRows(
 		sqlmock.NewRows([]string{"Table", "Create Table"}).
-			AddRow("test", "create table t(a int primary key, b int, c varchar(10))"))
+			AddRow("test", "create table t(a int primary key, b int, c varchar(10000), key(c))"))
```

Comment:
Should we test the maximum number of the `max-index-length` (i.e. `DefMaxOfMaxIndexLength` in TiDB) so that we can ensure we are compatible with create statements of any key length.

Reply:
now 20000 is larger than DefMaxOfMaxIndexLength 😄